utils/asset/post_scripts.py

```diff
@@ -400,7 +400,8 @@ def post_asset_types(f, file_length): # noqa
                 else:
                     asset = AssetType()
                     asset.asset_type = asset_type
-                    asset.asset_sub_category = sub_category_name
+                    asset.asset_sub_category = AssetSubCategory.objects\
```

Comment:
This query is redundant considering that we have an AssetSubCategory model object called `sub_category_name` already.
https://github.com/AndelaOSP/art-backend/blob/de8ce2b9824577cf4c5c5b95478efb3ae744e01d/utils/asset/post_scripts.py#L386-L387
Assigning `asset.asset_sub_category` to it is sufficient.
```python
asset.asset_sub_category = sub_category_name
```